Add support for M1 Macbook desktop testing

Building A-S for Android needs some updates to be out-of-the-box compatible with M1 Macbooks. I'm trying to work through the issues locally, but am new to the project so it's a bit slow going. I'll keep this issue updated with blockers and their solutions as I encounter them:

1. Android `sdkmanager` command line tools required by build not compatible with M1. The error I saw was `Could not find or load main class com.android.sdklib.tool.sdkmanager.SdkManagerCli`, but there may be others. See [this SO post](https://stackoverflow.com/questions/69869181/error-about-android-studio-on-macbook-m1-an-error-occurred-while-trying-to-comp?noredirect=1&lq=1).
  * Use Android Studio Canary version to download the required SDK tools. If done correctly, the executable will be found in: `Users/<user>/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager`.
1. Running `./libs/verify-android-environment.sh` fails with `Cannot get property 'nativeRustTarget' on extra properties extension as it does not exist`.
  * Add the following case to the nativeRustTarget switch:

```
switch (DefaultPlatform.RESOURCE_PREFIX) {
     case 'darwin':
     case 'darwin-x86-64':
+    case 'darwin-aarch64':
         ext.nativeRustTarget = 'darwin'
         break
     case 'linux-x86-64':
```

1. Re-running `./libs/verify-android-environment.sh` appears to have another error:

```
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

Traceback (most recent call last):
  File "/usr/local/bin/gyp", line 11, in <module>
    load_entry_point('gyp==0.1', 'console_scripts', 'gyp')()
  File "/Library/Python/2.7/site-packages/gyp-0.1-py2.7.egg/gyp/__init__.py", line 552, in script_main
    return main(sys.argv[1:])
  File "/Library/Python/2.7/site-packages/gyp-0.1-py2.7.egg/gyp/__init__.py", line 545, in main
    return gyp_main(args)
  File "/Library/Python/2.7/site-packages/gyp-0.1-py2.7.egg/gyp/__init__.py", line 530, in gyp_main
    generator.GenerateOutput(flat_list, targets, data, params)
  File "/Library/Python/2.7/site-packages/gyp-0.1-py2.7.egg/gyp/generator/ninja.py", line 2493, in GenerateOutput
    pool.map(CallGenerateOutputForConfig, arglists)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/pool.py", line 253, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/pool.py", line 572, in get
    raise self._value
AttributeError: 'NoneType' object has no attribute 'groups'
```

* This error can allegedly be ignored according to [this issue](https://github.com/nodejs/node-gyp/issues/569) and [this issue](https://github.com/nodejs/node-gyp/issues/1464)

1. Running `cargo test` causes the following error. This happens with different crates on subsequent runs:

    error: failed to run custom build command for `sql-support v0.1.0 (/Users/matthewtighe/dev/application-services/components/support/sql)`
    
    Caused by:
      process didn't exit successfully: `/Users/matthewtighe/dev/application-services/target/debug/build/sql-support-22f2ec6dbcbc1fcd/build-script-build` (exit status: 101)
      --- stdout
      cargo:rerun-if-changed=build.rs
      cargo:rerun-if-env-changed=NSS_DIR
    
      --- stderr
      thread 'main' panicked at 'It looks like NSS is not built. Please run `libs/verify-[platform]-environment.sh` first!', components/support/rc_crypto/nss/nss_build_common/src/lib.rs:55:9
      note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
    warning: build failed, waiting for other jobs to finish...
    error: build failedMy guess now is that `nss` is being compiled for `x86_64`  instead of `aarch64`, but I'm still trying to figure out how to check/change that.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-2904)
┆epic: local development, ci and release (backlog)
┆sprintEndDate: 2022-02-03

Looks like this issue is going to block me from a subset of A-C contributions as well. Trying to run [these tests](https://github.com/mozilla-mobile/android-components/blob/main/components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesHistoryStorageTest.kt) fails with the error at the bottom of this post. Looks like it's potentially the same runtime linking issue we ran into in [this post above](https://github.com/mozilla/application-services/issues/4665#issuecomment-975982143).

For context, the test suite in that class is instantiating a [PlacesApi](https://github.com/mozilla/application-services/blob/main/components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt) from A-S. 

```
Caused by: java.lang.UnsatisfiedLinkError: /Users/matthewtighe/Library/Caches/JNA/temp/jna11129591045367077596.tmp: dlopen(/Users/matthewtighe/Library/Caches/JNA/temp/jna11129591045367077596.tmp, 1): no suitable image found.  Did find:
	/Users/matthewtighe/Library/Caches/JNA/temp/jna11129591045367077596.tmp: no matching architecture in universal wrapper
	/Users/matthewtighe/Library/Caches/JNA/temp/jna11129591045367077596.tmp: no matching architecture in universal wrapper
	at java.base/java.lang.ClassLoader$NativeLibrary.load0(Native Method)
	at java.base/java.lang.ClassLoader$NativeLibrary.load(ClassLoader.java:2442)
	at java.base/java.lang.ClassLoader$NativeLibrary.loadLibrary(ClassLoader.java:2498)
	at java.base/java.lang.ClassLoader.loadLibrary0(ClassLoader.java:2694)
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2627)
	at java.base/java.lang.Runtime.load0(Runtime.java:768)
	at java.base/java.lang.System.load(System.java:1837)
	at com.sun.jna.Native.loadNativeDispatchLibraryFromClasspath(Native.java:1018)
	at com.sun.jna.Native.loadNativeDispatchLibrary(Native.java:988)
	at com.sun.jna.Native.<clinit>(Native.java:195)
	at com.sun.jna.Structure.setAlignType(Structure.java:280)
	at com.sun.jna.Structure.<init>(Structure.java:197)
	at com.sun.jna.Structure.<init>(Structure.java:193)
	at com.sun.jna.Structure.<init>(Structure.java:180)
	at com.sun.jna.Structure.<init>(Structure.java:172)
	at mozilla.appservices.places.RustError.<init>(RustError.kt:15)
	at mozilla.appservices.places.RustError$ByReference.<init>(RustError.kt:17)
	at mozilla.appservices.places.PlacesApi.<init>(PlacesConnection.kt:1543)
	at mozilla.components.browser.storage.sync.RustPlacesConnection.init(Connection.kt:79)
	at mozilla.components.browser.storage.sync.PlacesStorage$places$2.invoke(PlacesStorage.kt:48)
	at mozilla.components.browser.storage.sync.PlacesStorage$places$2.invoke(PlacesStorage.kt:47)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at mozilla.components.browser.storage.sync.PlacesStorage.getPlaces$browser_storage_sync_debug(PlacesStorage.kt:47)
	at mozilla.components.browser.storage.sync.PlacesHistoryStorage$deleteEverything$2.invokeSuspend(PlacesHistoryStorage.kt:159)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
```

--- a/build.gradle
+++ b/build.gradle
@@ -21,7 +21,7 @@ buildscript {
         mockito_core_version = '2.28.2'
         roboelectric_core_version = '4.5.1'
         robolectric_core_version = '4.5.1'
-        rust_android_gradle_version = '0.8.3'
+        rust_android_gradle_version = '0.9.1'
         espresso_core_version = '3.3.0'
         protobuf_version = '3.11.4'
         gradle_protobuf_version = '0.8.14'
@@ -61,7 +61,7 @@ buildscript {
         // Publish.
         classpath "digital.wup:android-maven-publish:$maven_publish_version"
 
-        classpath "gradle.plugin.org.mozilla.rust-android-gradle:plugin:$rust_android_gradle_version"
+        classpath "org.mozilla.rust-android-gradle:plugin:$rust_android_gradle_version"
 
         // Yes, this is unusual.  We want to access some host-specific
         // computation at build time.
@@ -145,8 +145,8 @@ if (useDownloadedLibs) {
     task downloadDesktopLibs(type: Download) {
         src {
             switch (DefaultPlatform.RESOURCE_PREFIX) {
-                case 'darwin':
                 case 'darwin-x86-64':
+                case 'darwin-aarch64':
                     return "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/project.application-services.application-services.build.libs.desktop.macos.${rootProject.ext.libsGitSha}/artifacts/public/build/macos.tar.gz"
                 case 'linux-x86-64':
                     return "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/project.application-services.application-services.build.libs.desktop.linux.${rootProject.ext.libsGitSha}/artifacts/public/build/linux.tar.gz"
@@ -162,8 +162,8 @@ if (useDownloadedLibs) {
 
         dest {
             switch (DefaultPlatform.RESOURCE_PREFIX) {
-                case 'darwin':
                 case 'darwin-x86-64':
+                case 'darwin-aarch64':
                     return new File(buildDir, "libs.desktop.macos.${rootProject.ext.libsGitSha}.tar.gz")
                 case 'linux-x86-64':
                     return new File(buildDir, "libs.desktop.linux.${rootProject.ext.libsGitSha}.tar.gz")
@@ -229,10 +229,11 @@ ext.rustTargets = [
 
 // Generate libs for our current platform so we can run unit tests.
 switch (DefaultPlatform.RESOURCE_PREFIX) {
-    case 'darwin':
     case 'darwin-x86-64':
+        ext.nativeRustTarget = 'darwin-x86-64'
+        break
     case 'darwin-aarch64':
-        ext.nativeRustTarget = 'darwin'
+        ext.nativeRustTarget = 'darwin-aarch64'
         break
     case 'linux-x86-64':
         ext.nativeRustTarget = 'linux-x86-64'

--- a/build.gradle
+++ b/build.gradle
@@ -231,6 +231,7 @@ ext.rustTargets = [
 switch (DefaultPlatform.RESOURCE_PREFIX) {
     case 'darwin':
     case 'darwin-x86-64':
+    case 'darwin-aarch64':
         ext.nativeRustTarget = 'darwin'
         break
     case 'linux-x86-64':

--- a/libs/build-nss-desktop.sh
+++ b/libs/build-nss-desktop.sh
@@ -34,12 +34,13 @@ elif [[ -n "${CROSS_COMPILE_TARGET}" ]]; then
   echo "Cannot build NSS for unrecognized target OS ${CROSS_COMPILE_TARGET}"
   exit 1
 elif [[ "$(uname -s)" == "Darwin" ]]; then
-  DIST_DIR=$(abspath "desktop/darwin/nss")
   TARGET_OS="macos"
   # We need to set this variable for switching libs based on different macos archs (M1 vs Intel)
   if [[ "$(uname -m)" == "arm64" ]]; then
+    DIST_DIR=$(abspath "desktop/darwin-aarch64/nss")
     TARGET_ARCH="aarch64"
   else
+    DIST_DIR=$(abspath "desktop/darwin-x86-64/nss")
     TARGET_ARCH="x86_64"
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then

--- a/libs/build-nss-desktop.sh
+++ b/libs/build-nss-desktop.sh
@@ -40,7 +40,7 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
     DIST_DIR=$(abspath "desktop/darwin-aarch64/nss")
     TARGET_ARCH="aarch64"
   else
-    DIST_DIR=$(abspath "desktop/darwin_x86-64/nss")
+    DIST_DIR=$(abspath "desktop/darwin-x86_64/nss")
     TARGET_ARCH="x86_64"
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then

--- a/libs/build-nss-desktop.sh
+++ b/libs/build-nss-desktop.sh
@@ -40,7 +40,7 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
     DIST_DIR=$(abspath "desktop/darwin-aarch64/nss")
     TARGET_ARCH="aarch64"
   else
-    DIST_DIR=$(abspath "desktop/darwin-x86-64/nss")
+    DIST_DIR=$(abspath "desktop/darwin_x86-64/nss")
     TARGET_ARCH="x86_64"
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then

--- a/libs/build-nss-desktop.sh
+++ b/libs/build-nss-desktop.sh
@@ -40,7 +40,7 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
     DIST_DIR=$(abspath "desktop/darwin-aarch64/nss")
     TARGET_ARCH="aarch64"
   else
-    DIST_DIR=$(abspath "desktop/darwin-x86_64/nss")
+    DIST_DIR=$(abspath "desktop/darwin-x86-64/nss")
     TARGET_ARCH="x86_64"
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then

--- a/libs/build-sqlcipher-desktop.sh
+++ b/libs/build-sqlcipher-desktop.sh
@@ -42,8 +42,8 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
     NSS_DIR=$(abspath "desktop/darwin-aarch64/nss")
   else
     TARGET_ARCH="x86_64"
-    DIST_DIR=$(abspath "desktop/darwin_x86-64/sqlcipher")
-    NSS_DIR=$(abspath "desktop/darwin_x86-64/nss")
+    DIST_DIR=$(abspath "desktop/darwin-x86_64/sqlcipher")
+    NSS_DIR=$(abspath "desktop/darwin-x86_64/nss")
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then
   # This is a JNA weirdness: "x86-64" rather than "x86_64".

--- a/libs/build-sqlcipher-desktop.sh
+++ b/libs/build-sqlcipher-desktop.sh
@@ -34,14 +34,16 @@ elif [[ -n "${CROSS_COMPILE_TARGET}" ]]; then
   echo "Cannot build SQLCipher for unrecognized target OS ${CROSS_COMPILE_TARGET}"
   exit 1
 elif [[ "$(uname -s)" == "Darwin" ]]; then
-  DIST_DIR=$(abspath "desktop/darwin/sqlcipher")
-  NSS_DIR=$(abspath "desktop/darwin/nss")
   TARGET_OS="macos"
   # We need to set this variable for switching libs based on different macos archs (M1 vs Intel)
   if [[ "$(uname -m)" == "arm64" ]]; then
     TARGET_ARCH="aarch64"
+    DIST_DIR=$(abspath "desktop/darwin-aarch64/sqlcipher")
+    NSS_DIR=$(abspath "desktop/darwin-aarch64/nss")
   else
     TARGET_ARCH="x86_64"
+    DIST_DIR=$(abspath "desktop/darwin-x86-64/sqlcipher")
+    NSS_DIR=$(abspath "desktop/darwin-x86-64/nss")
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then
   # This is a JNA weirdness: "x86-64" rather than "x86_64".

--- a/libs/build-sqlcipher-desktop.sh
+++ b/libs/build-sqlcipher-desktop.sh
@@ -42,8 +42,8 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
     NSS_DIR=$(abspath "desktop/darwin-aarch64/nss")
   else
     TARGET_ARCH="x86_64"
-    DIST_DIR=$(abspath "desktop/darwin-x86-64/sqlcipher")
-    NSS_DIR=$(abspath "desktop/darwin-x86-64/nss")
+    DIST_DIR=$(abspath "desktop/darwin_x86-64/sqlcipher")
+    NSS_DIR=$(abspath "desktop/darwin_x86-64/nss")
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then
   # This is a JNA weirdness: "x86-64" rather than "x86_64".

--- a/libs/build-sqlcipher-desktop.sh
+++ b/libs/build-sqlcipher-desktop.sh
@@ -42,8 +42,8 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
     NSS_DIR=$(abspath "desktop/darwin-aarch64/nss")
   else
     TARGET_ARCH="x86_64"
-    DIST_DIR=$(abspath "desktop/darwin-x86_64/sqlcipher")
-    NSS_DIR=$(abspath "desktop/darwin-x86_64/nss")
+    DIST_DIR=$(abspath "desktop/darwin-x86-64/sqlcipher")
+    NSS_DIR=$(abspath "desktop/darwin-x86-64/nss")
   fi
 elif [[ "$(uname -s)" == "Linux" ]]; then
   # This is a JNA weirdness: "x86-64" rather than "x86_64".

--- a/libs/verify-desktop-environment.sh
+++ b/libs/verify-desktop-environment.sh
@@ -6,6 +6,9 @@
 
 set -e
 
+yellow=`tput setaf 3`
+reset=`tput sgr0`
+
 if [[ ! -f "$(pwd)/libs/build-all.sh" ]]; then
   echo "ERROR: bootstrap-desktop.sh should be run from the root directory of the repo"
   exit 1
@@ -14,7 +17,11 @@ fi
 "$(pwd)/libs/verify-common.sh"
 
 if [[ "$(uname -s)" == "Darwin" ]]; then
-  APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin"
+  if [[ "$(uname -m)" == "arm64" ]]; then
+    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-aarch64"
+  else
+    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-x86-64"
+  fi
 else
   APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/linux-x86-64"
 fi
@@ -22,6 +29,20 @@ fi
 if [[ -z "${SQLCIPHER_LIB_DIR}" ]] || [[ -z "${SQLCIPHER_INCLUDE_DIR}" ]] || [[ -z "${NSS_DIR}" ]] || [[ -z "${NSS_STATIC}" ]]; then
   echo "Some necessary environment variables are not set."
   echo "Please export or add to your shell initialization file (.zshenv, .bashrc etc.) the following:"
+  echo ""
+  echo "export SQLCIPHER_LIB_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/lib"
+  echo "export SQLCIPHER_INCLUDE_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/include"
+  echo "export NSS_DIR=${APPSERVICES_PLATFORM_DIR}/nss"
+  echo "export NSS_STATIC=1"
+  exit 1
+fi
+
+# If users previously have built NSS their env vars will still be pointing to darwin
+# we need to tell them to update their env with the new arch-specific style
+if [[ "${NSS_DIR}" != *"desktop/darwin-"*  ]]; then
+  echo ""
+  echo "${yellow}Your environment variables are outdated! Please use the updated values below"
+  echo "Please export or add to your shell initialization file (.zshenv, .bashrc etc.) the following:${reset}"
   echo ""
   echo "export SQLCIPHER_LIB_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/lib"
   echo "export SQLCIPHER_INCLUDE_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/include"

--- a/libs/verify-desktop-environment.sh
+++ b/libs/verify-desktop-environment.sh
@@ -20,7 +20,7 @@ if [[ "$(uname -s)" == "Darwin" ]]; then
   if [[ "$(uname -m)" == "arm64" ]]; then
     APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-aarch64"
   else
-    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin_x86-64"
+    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-x86_64"
   fi
 else
   APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/linux-x86-64"

--- a/libs/verify-desktop-environment.sh
+++ b/libs/verify-desktop-environment.sh
@@ -6,6 +6,9 @@
 
 set -e
 
+YELLOW=$"\033[1;33m"
+NC=$"\033[0m" # No Color
+
 if [[ ! -f "$(pwd)/libs/build-all.sh" ]]; then
   echo "ERROR: bootstrap-desktop.sh should be run from the root directory of the repo"
   exit 1
@@ -36,10 +39,10 @@ fi
 
 # If users previously have built NSS their env vars will still be pointing to darwin
 # we need to tell them to update their env with the new arch-specific style
-if [[ "${NSS_DIR}" != *"desktop/darwin-"*  ]]; then
+if [[ -z "${CI}" ]] && [[ "$(uname -s)" == "Darwin" ]] && [[ "${NSS_DIR}" != *"desktop/darwin-"*  ]]; then
   echo ""
-  echo "!! Your environment variables are outdated! Please use the updated values below !!"
-  echo "Please export or add to your shell initialization file (.zshenv, .bashrc etc.) the following"
+  echo -e "${YELLOW}!! Your environment variables are outdated! Please use the updated values below !!"
+  echo -e "Please export or add to your shell initialization file (.zshenv, .bashrc etc.) the following ${NC}"
   echo ""
   echo "export SQLCIPHER_LIB_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/lib"
   echo "export SQLCIPHER_INCLUDE_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/include"

--- a/libs/verify-desktop-environment.sh
+++ b/libs/verify-desktop-environment.sh
@@ -20,7 +20,7 @@ if [[ "$(uname -s)" == "Darwin" ]]; then
   if [[ "$(uname -m)" == "arm64" ]]; then
     APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-aarch64"
   else
-    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-x86-64"
+    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin_x86-64"
   fi
 else
   APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/linux-x86-64"

--- a/libs/verify-desktop-environment.sh
+++ b/libs/verify-desktop-environment.sh
@@ -6,9 +6,6 @@
 
 set -e
 
-yellow=`tput setaf 3`
-reset=`tput sgr0`
-
 if [[ ! -f "$(pwd)/libs/build-all.sh" ]]; then
   echo "ERROR: bootstrap-desktop.sh should be run from the root directory of the repo"
   exit 1
@@ -20,7 +17,7 @@ if [[ "$(uname -s)" == "Darwin" ]]; then
   if [[ "$(uname -m)" == "arm64" ]]; then
     APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-aarch64"
   else
-    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-x86_64"
+    APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/darwin-x86-64"
   fi
 else
   APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/linux-x86-64"
@@ -41,8 +38,8 @@ fi
 # we need to tell them to update their env with the new arch-specific style
 if [[ "${NSS_DIR}" != *"desktop/darwin-"*  ]]; then
   echo ""
-  echo "${yellow}Your environment variables are outdated! Please use the updated values below"
-  echo "Please export or add to your shell initialization file (.zshenv, .bashrc etc.) the following:${reset}"
+  echo "!! Your environment variables are outdated! Please use the updated values below !!"
+  echo "Please export or add to your shell initialization file (.zshenv, .bashrc etc.) the following"
   echo ""
   echo "export SQLCIPHER_LIB_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/lib"
   echo "export SQLCIPHER_INCLUDE_DIR=${APPSERVICES_PLATFORM_DIR}/sqlcipher/include"

